zschema/compounds.py

```diff
@@ -10,6 +11,18 @@ def _is_valid_object(name, object_):
     if not isinstance(object_, Keyable):
         raise Exception("Invalid schema. %s is not a Keyable." % name)
 
+def _proto_message_name(string):
+    if string != string.lower():
+        return string
+    string = "".join(w.capitalize() for w in string.split("_"))
+    string = string.replace("Ztag", "ZTag")
```

Comment:
This scares me a bit. Where does ZTag show up in current schemas? Is it only ZTag? Does ZGrab show up?

Reply:
I'm going to remove this.